### PR TITLE
Add notes

### PR DESCRIPTION
**Pull request by Ldhatter**
#30 Add Notes

User can add a note to a participant profile.
This includes text and an upload image of size < 40KB

### CODE REVIEW

File: angular-client/src/app/components/participant-profile/participant-profile.component.ts [hunk 1]
```diff
@@ -36,13 +43,30 @@ export class ParticipantProfileComponent implements OnInit {
...
   }
+
+  /**
+   * Add a note to a participant
+   * 
+   * @memberof ParticipantProfileComponent
+   */
+  addNote(): void {
+    let dialogRef = this.dialog.open(NoteComponent, {
+      width: '66%',
```
Comment (annarog) [inline angular-client/src/app/components/participant-profile/participant-profile.component.ts]: is there a particular reason why it's here and not in the CSS?

File: angular-client/src/app/components/note/note.component.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      console.log(data);
+      this.dialogRef.close();
+    });
+  }
+
+  cancel(): void {
+    this.dialogRef.close();
+  }
+
+  handleFileInput(files) {
```
Comment (annarog) [inline angular-client/src/app/components/note/note.component.ts]: Document this file :)

File: angular-client/src/app/components/participant-profile/participant-profile.component.ts (hunk 1 above)
Comment (Ldhatter) [reply angular-client/src/app/components/participant-profile/participant-profile.component.ts]: to specify the width upon creating it. if not there's currently no way to get the parent div by using only the css once it's created.

File: angular-client/src/app/components/note/note.component.ts (hunk 2 above)
Comment (Ldhatter) [reply angular-client/src/app/components/note/note.component.ts]: Ok completed with commit a7d968a

File: angular-client/src/app/components/participant-profile/participant-profile.component.ts (hunk 1 above)
Comment (annarog) [reply angular-client/src/app/components/participant-profile/participant-profile.component.ts]: got it 👍

File: angular-client/src/app/components/note/note.component.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+@Component({
+  selector: 'app-note',
+  templateUrl: './note.component.html',
+  styleUrls: ['./note.component.css']
+})
+export class NoteComponent implements OnInit {
+
+  note: Note = {
+    text: "",
```
Comment (adriannadiaz) [inline angular-client/src/app/components/note/note.component.ts]: single quotes :)

File: angular-client/src/app/services/participant.service.ts [hunk 4]
```diff
@@ -93,7 +110,7 @@ export class ParticipantService {
   search(participantAttributeValuePair): Observable<Object> {
     return this.http.get(`${this.url}/search/${participantAttributeValuePair}`)
       .pipe(
-      map(participants => participants[0]? true: false),
+      map(participants => participants[0] ? true : false),
```
Comment (adriannadiaz) [inline angular-client/src/app/services/participant.service.ts]: Avoid the ternary. It doesn't pass linting...I know you love it but it leads to 🐛

Comment (Ldhatter) [reply angular-client/src/app/services/participant.service.ts]: my linter does not complain about ternaries (which are part of js specs, and therefore part of ts). does your linter provide any reasoning to why it doesn't like it? if not, can you customize yours to allow ternaries. thanks

File: angular-client/src/app/components/note/note.component.ts (hunk 3 above)
Comment (Ldhatter) [reply angular-client/src/app/components/note/note.component.ts]: oh which style are we going for @adriannadiaz?
double quotes - [typescript](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#strings)
or single quotes - which guide?

File: angular-client/src/app/services/participant.service.ts (hunk 4 above)
Comment (adriannadiaz) [reply angular-client/src/app/services/participant.service.ts]: Fair enough.

File: angular-client/src/app/components/note/note.component.ts (hunk 3 above)
Comment (adriannadiaz) [reply angular-client/src/app/components/note/note.component.ts]: My internal linter says ' quotes

**Pull request merged**